### ISSUE TITLE
Endpoint for authenticated subject alias

### ISSUE THREAD

**Issue by leodido** [fabbricadigitale/scimd#68]
How to handle the authenticated subject alias (eg., `/Me` endpoint) - Section 3.11 @ RFC7644
- [ ] Permanent redirect (308) to `/User`
- [ ] Process directly with `Location` header pointing to the aliased resource (ie., `User`)
- [ ] Both way but configurable

Copying from my notes at PR #46.

cc @fabbricadigitale/openinnovation

**Comment by leogr** [comment]
`/Me` can work only if an authentication mean has been implemented by service provider using `User` as identity storage. Currently it's not.

It depends on #65 so I'm just proposing to postpone this feature.

**Comment by leodido** [comment]
I too, was only for future reference.
